llvm/docs/SecurityTransparencyReports.rst

```diff
@@ -2,7 +2,15 @@
 LLVM Security Group Transparency Reports
 ========================================
 
-This page lists the yearly LLVM Security group transparency reports.
+This page lists the yearly LLVM Security Response group transparency reports.
+
+The LLVM Security Response group started out as the LLVM security group, previous
+year's transparency reports keep the original year.
+
+Initially the Chromium issue tracker was used to record issues. This component
+has been archived and is read-only. A llvm-project issue with the SecurityArchive
+label has been created with a further backup PDF copy in a llvm-project
+repository.
```

Comment:
It took me time to understand that this actually means "Each of the original  issue from the chromium issue tracker has been mirrored as an llvm-project issue that keeps a link to a further backup PDF copy stored as an issue attachment." (feel free to rephrase, but I bugged on "with a further backup PDF copy in a llvm-project
repository")